### ISSUE TITLE
Due 2/12: Open Government Partnership call for innovative open government cases proposal

### ISSUE THREAD

**Issue by mcharg** [ONRR/doi-extractives-data#3572]
The OECD and [OGP](https://www.opengovpartnership.org/) are launching a call for open government innovations. Submit your case and you could be featured in upcoming OECD Open Government Reviews and be amongst the two case authors to be invited to the OECD in Paris in 2019.

The deadline for submissions: 12 February 2019

Open government innovators from around the world are encouraged to submit innovative practices to propel research on open government trends and to inspire other members of the community to work in new and creative ways. Be part of this vibrant community, paving the way for a new wave of open government.

What is an open government innovation?

An open government innovation is any good practice that promotes the principles of transparency, integrity, accountability and stakeholder participation in support of democracy and inclusive growth.  

  At its core, an open government innovation is:

Are you innovating to find solutions that promote the principles of transparency, integrity, accountability or stakeholder participation and push the open government agenda forward? Do you work on civic participation? Are you a leader in making government more transparent and accountable?  Is your goal to shine a light on how public finances are used? Are you aware of an approach to build open government skills among public officials?

Then share your story with us! We want to hear from you – in government, civil society or the private sector – on how you address challenges in your community through innovative open government practices. We welcome submissions in English, French and Spanish.

How to Submit an Open Government Innovation:

1.	First, register to the [OPSI Website](https://oecd-opsi.org/register/) to submit your case and become part of the OECD Open Government community of practice.

2. Make sure you fill in the [submission criteria](https://oecd-opsi.org/criteria-for-acceptance-2/) for acceptance:
- The case is relevant to the open government principles of transparency, integrity, accountability and stakeholder participation.
- It has a novel approach, either as a concept, or an upgrade based on a previous initiative. It is ongoing or has been implemented in practice, and it has shown results that can be verified. 
-The case study is written in plain English or French.
-Every section of the case submission form is completed online.

3.	Then, submit an Innovation through our [platform](https://oecd-opsi.org/case-study-form-open-government/) by 12 February 2019.

You can preview the submission questions and share them with your team using the [.doc version](https://oecd-opsi.org/wp-content/uploads/2018/11/Open-Government-Case-Study-Submission-Form.doc) available here. Please note that final submissions will need to be done through the digital platform. The OECD Open Government team and peer reviewers can suggest edits to the case description. The final selection of cases to be published is chosen by the OECD team, as advised by members of an informal Expert Group on Open Government.

**Comment by mcharg** [comment]
@jennmalcolm This one seems like a better fit for us with potential broader outreach potential than the summit and the deadline is slightly farther out.

**Comment by jennmalcolm** [comment]
This does look interesting. Let me do a little research to see how difficult it would be to get approval for this since it is also international.

**Comment by mcharg** [comment]
Here are some sample case studies that have already been published. https://oecd-opsi.org/case_type/open-government/

**Comment by mcharg** [comment]
Based on their definition of innovation, our case study should probably be about shifting from EITI to a user-centered model.
https://oecd-opsi.org/explore-what-an-innovation-is-and-is-not-2/

**Comment by mcharg** [comment]
@jennmalcolm this is ready for you to review.
https://docs.google.com/document/d/1f-9Cl3k5x4UZho2IQ_fEFbP5bWzGHZg9m_HgIoQNwnI/edit

**Comment by jennmalcolm** [comment]
I have reviewed it and put my comments in the document.

**Comment by mcharg** [comment]
I started filling out the submission form, but ran into issues uploading images.  Their support said to email the images.  @lpgoldstein Do you want to take one last pass through the doc before I submit?

**Comment by usEITI** [comment]
It looks good to me.  I like it!

**Comment by mcharg** [comment]
They finally published our case study!

https://oecd-opsi.org/innovations/redesigning-the-u-s-natural-resources-revenue-data-nrrd-portal-homepage-to-meet-user-needs/
